node-launchpad/src/app.rs

```diff
@@ -46,39 +52,61 @@ impl App {
         peers_args: PeersArgs,
         safenode_path: Option<PathBuf>,
     ) -> Result<Self> {
+        // Configurations
         let app_data = AppData::load()?;
+        let config = Config::new()?;
+
+        let data_dir_path = match &app_data.storage_mountpoint {
+            Some(path) => get_launchpad_nodes_data_dir_path(PathBuf::from(path))?,
+            None => return Err(eyre!("Storage mountpoint for node data is not set")),
+        };
+        debug!("Data dir path for nodes: {data_dir_path:?}");
 
-        let home = Home::new(
+        // Main Screens
+        let status = Status::new(
             app_data.nodes_to_start,
             &app_data.discord_username,
             peers_args,
             safenode_path,
+            data_dir_path,
         )
         .await?;
-        let config = Config::new()?;
-        let discord_username_input = BetaProgramme::new(app_data.discord_username.clone());
-        let manage_nodes = ManageNodes::new(app_data.nodes_to_start)?;
-        let footer = Footer::new(app_data.nodes_to_start > 0);
-        let help = HelpPopUp::default();
+        let options = Options::new(
+            app_data.storage_mountpoint.clone().unwrap(),
+            app_data.storage_drive.clone().unwrap(),
```

Comment:
Can we use the `data_dir_path` that is obtained above? eyre will throw a nice error msgs than using unwrap().

Also we could do the same error handling for `app_data.storage_drive`?

Reply:
`data_dir_path` is the mountpoint + the directory where to store nodes data. I would prefer to store the mountpoint instead of the actual directory as we are storing drive related info (name and mountpoint).

Regarding `eyre`, I can use something like:

`app_data.storage_mountpoint.clone().ok_or_else(|| eyre::eyre!("storage_mountpoint is None"))?,`

wdyt?

Reply:
> data_dir_path is the mountpoint + the directory where to store nodes data. 
Yeah, sorry misread it.

And I like the eyre error! We can go with that!